AnnoDesigner/Localization/TranslateExtension.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Windows.Data;
+
+namespace AnnoDesigner.Localization
+{
+    public class TranslateKeyConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            var translations = value as IDictionary<string, string>;
+            var key = parameter as string;
```

Comment:
I think we need a null check for `parameter` (or `key`) here - `ContainsKey()` throws a `NullReferenceException` if `key` is null, which could occur if a developer does not add the key in the XAML.

If we wanted to keep this behaviour intentionally, we should probably do the same thing as suggested in the comment below and get it to throw only in Debug mode, and just return some other value such as an empty string when in a release build.